src/internal/cmdutil/cobra.go

```diff
@@ -337,6 +347,11 @@ func CreateAlias(cmd *cobra.Command, invocation string) *cobra.Command {
 				cur.Use = strings.ReplaceAll(cmd.Use, "{{alias}}", arg)
 			}
 			cur.Example = strings.ReplaceAll(cmd.Example, "{{alias}}", fmt.Sprintf("%s %s", os.Args[0], invocation))
+
+			if len(plurals) != 0 {
+				cur.Aliases = append([]string{cur.Use}, plurals...)
```

Comment:
does cur.Use need to be re-included as an alias here?

Reply:
It doesn't actually, and it probably shouldn't. I think doing so actually makes it show up twice in the aliases section:

```
Aliases:
  repo, repo, repos
```

So I can remove it and that should hopefully clean up the aliases section too